documentation/man/kw.rst

```diff
@@ -101,16 +101,15 @@ This command is a wrapper for **checkpatch**, with the goal of simplifying the
 use of this tool; notice that you can specify a single file or an entire
 directory.
 
-e, explore [log] [*EXPRESSION*] [-p] [*DIRECTORY|FILE*]
+e, explore [--log] [*EXPRESSION*] [-p] [*DIRECTORY|FILE*]
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The *explore* command is based on git grep. It can search for string matches in
```

Comment:
While we are here, what do you think of changing this line to "*explore* command is a wrapper to git grep", instead of "based on" (which gives the idea that we re-implemented the search)?